--- origsrc/qmmp-0.7.4/src/qmmp/CMakeLists.txt	2013-12-24 00:29:48.000000000 -0600
+++ src/qmmp-0.7.4/src/qmmp/CMakeLists.txt	2014-02-12 01:16:14.859291900 -0600
@@ -14,7 +14,9 @@ ADD_DEFINITIONS(-DQT_NO_DEBUG)
 ADD_DEFINITIONS(-DQT_SHARED)
 ADD_DEFINITIONS(-DQT_THREAD)
 ADD_DEFINITIONS(-DLIB_DIR=\\\"\\/${LIB_DIR}\\\")
+IF(NOT WIN32)
 ADD_DEFINITIONS(-DQMMP_INSTALL_PREFIX=\\\"${CMAKE_INSTALL_PREFIX}\\\")
+ENDIF(NOT WIN32)
 
 IF(NOT QMMP_VERSION_STABLE) #add svn revision
 execute_process(COMMAND ${CMAKE_CURRENT_SOURCE_DIR}/svn_revision.sh
--- origsrc/qmmp-0.7.4/src/qmmp/qmmp.cpp	2013-12-24 00:29:48.000000000 -0600
+++ src/qmmp-0.7.4/src/qmmp/qmmp.cpp	2014-02-12 01:17:47.777606500 -0600
@@ -70,7 +70,7 @@ const QString Qmmp::pluginsPath()
     QDir dir(QMMP_INSTALL_PREFIX"/"LIB_DIR"/qmmp");
     //qDebug(QMMP_INSTALL_PREFIX"/"LIB_DIR"/qmmp");
 #else
-#if defined(Q_OS_WIN32) && !defined(Q_OS_CYGWIN)
+#if defined(Q_OS_WIN32) && !defined(Q_CC_MINGW) && !defined(Q_OS_CYGWIN)
     QDir dir(qApp->applicationDirPath() + "/plugins");
 #else
     QDir dir(qApp->applicationDirPath() + "/../"LIB_DIR"/qmmp");
